CMakeLists.txt

```diff
@@ -124,20 +146,29 @@ if(ENABLE_TCMALLOC AND HAVE_LIBTCMALLOC)
     target_link_libraries(wiredtiger ${HAVE_LIBTCMALLOC})
 endif()
 
+# When handling builtins, its still ideal to call 'target_link_libraries'
```

Comment:
Might want to reword this comment. We _were_ calling `target_link_libraries` on the builtin library targets to capture their transitive dependencies. Though since we changed it, probably just want to reword the comment to say we are explicitly linking the 3rd party libraries each builtin depends on.

Reply:
done! let me know if you think how the new comment is.